concourse/pipeline.yml

```diff
@@ -1640,7 +1640,7 @@ jobs:
           args:
           - -exc
           - |
-            mvn --settings rm-sdx-gateway-source/.travis.settings.xml install -Ddockerfile.skip -f rm-sdx-gateway-source/pom.xml
+            mvn --settings rm-sdx-gateway-source/.maven.settings.xml install -Ddockerfile.skip -f rm-sdx-gateway-source/pom.xml
```

Comment:
Looks like `.travis.settings.xml` is still present in this repo.

Reply:
Think this is ok now